src/main/java/org/commonjava/indy/service/tracking/change/FoloTrackingListener.java

```diff
@@ -75,7 +75,10 @@ public void handleFileAccessEvent( final FileEvent event )
 
         String trackingId = event.getSessionId();
 
-        final TrackingKey trackingKey = !Objects.equals( trackingId, "" ) ? new TrackingKey( trackingId ) : null;
+        TrackingKey trackingKey = null;
+        if (!Objects.equals( trackingId, "" ) && trackingId != null) {
```

Comment:
Can we use StringUtils.isNotBlank (in commons-lang3)  instead here?

Reply:
updated to use isNotBlank. :)